source/controllers/representing-multiple-models.md

```diff
@@ -53,7 +50,7 @@ Now we can use this property in our template:
 
 ### Representing Several Models
 
-Multiple models can be returned through `Ember.RSVP.hash`. We can return a hash from the `model` hook, but `Ember.RSVP.hash` takes parameters that return promises and when all parameter promises resolve then the `Ember.RSVP.hash` promise resolves. To continue with the above example, if we also wanted to return the `model:album` records then our `model` hook would look like:
+Multiple models can be returned through [Ember.RSVP.hash](http://emberjs.com/api/classes/RSVP.html#method_hash). We can return a hash from the `model` hook, but `Ember.RSVP.hash` takes parameters that return promises and when all parameter promises resolve then the `Ember.RSVP.hash` promise resolves. To continue with the above example, if we also wanted to return the `model:album` records then our `model` hook would look like:
```

Comment:
line breaks, plz.
can you change `above example` to `example above`? I think it's still correct and less confusing.